src/Uno.UI.Tests/Windows_UI_Xaml/Given_ThemeResource.cs

```diff
@@ -14,14 +14,35 @@ namespace Uno.UI.Tests.Windows_UI_Xaml
 	public class Given_ThemeResource
 	{
 		[TestMethod]
-		public void When_System_ThemeResource()
+		public void When_System_ThemeResource_Light()
 		{
 			var app = UnitTestsApp.App.EnsureApplication();
+			app.ForceSetRequestedTheme(Windows.UI.Xaml.ApplicationTheme.Light);
```

Comment:
Should there be a reset of the theme as well here ? (even if it's the default)

Reply:
Fixed :-)